app/containers/TextInput/FormTextInput.tsx

```diff
@@ -115,7 +115,9 @@ export const FormTextInput = ({
 			<A11yElement order={1}>
 				<View
 					accessible
-					accessibilityLabel={`${label} ${accessibilityLabelRequired} ${accessibilityInputValue}. ${accessibilityInputError}`}
+					accessibilityLabel={`${
+						accessibilityLabel ?? label
```

Comment:
It's not going to change much, but I'm going to mention since I'm here :)
`??` comparison is going to make `label` be used only if `accessibilityLabel` is `null` or `undefined`.
So if it's `''`, `label` won't be used.
That's probably not expected, so `||` comparison is better on this case.

![image](https://github.com/user-attachments/assets/bef171cb-19e1-4cbd-bd4f-fc2ac069ff5d)